setup.cfg

```diff
@@ -36,7 +36,14 @@ install_requires=
     cwlprov==0.1.1
     networkx==2.8
     prov>=1.5.1
-    rocrate~=0.7
+    rocrate~=0.7    
+    rdflib~=6.2
+    PyShEx~=0.8.1
+    arcp~=0.2.1
+    rdflib_shim>=1.0.3
+    rocrateValidator==0.2.15
+    pytest # FIXME: strangely needed by rocrateValidator
```

Comment:
`pytest` is imported by a module shipped with the wheel/sdist installed from pypi, even though I believe it's not used: https://github.com/ResearchObject/ro-crate-validator-py/issues/4